src/pages/tBTC/Bridge/Unmint.tsx

```diff
@@ -73,11 +72,7 @@ const UnmintFormPage: PageComponent = ({}) => {
 
   return (
     <>
-      <BridgeProcessCardTitle bridgeProcess="unmint" />
-      <BridgeProcessCardSubTitle
-        stepText="Step 1"
-        subTitle="Unmint your tBTC tokens"
-      />
+      <BridgeProcessCardTitle step={1} title="Unmint your tBTC tokens" />
```

Comment:
It just popped out to me now, that `BridgeProcessCardTitle` is not reusable enough to use it in `Unmint` or `ResumeDeposit`. As of now it looks like we don't have the designs for those, but we will probably implement `Unmint` (not sure about `ResumeDeposit`). For `Unmint` though, the TOTAL_STEPS count will not be the same as in minting flow, so I think that we should either (from worst to best, pick one, I recommend picking 3):

1. Rename `BridgeProcessCardTitle` to `MintingProcessCardTitle`, but I fell like we will want to use some of it's code to Unmint and ResumeDeposit in the future. 
2. Make `steps` an optional property and allow to add `totalSteps` optionally too. We will display the `{step}/{TOTAL_STEPS}` thing only when `steps` AND `totalSteps` are provided
3. (this one would be the best IMO) Forget about `steps` / `totalSteps` whatsoever and create a new property called `badgeText`. We can then pass the steps and total steps as string there, for example:

```
<BridgeProcessCardTitle badgeText={1/3} title="Unmint your tBTC tokens" />
```

I think this one would be the best because this way, the `BridgeProcessCardTitle` will be more reusable. If we provide `badgeText` it will display the badge with the text on the left side of the title.

Another good apporach would be to create a beforeText prop (similar to `afterDescription` and pass a Badge with steps there. But it looks like this badge is the common thing for those Bridge Process Card Titles and we don't plan to display anything else there, so I would just create `badgeText` for now.

Reply:
Ref commit: 5e281add

Reply:
Looking good, although ideally we would want to make `badgeText` optional like I mentioned here:

> If we provide titleBadgeText it will display the badge with the text on the left side of the title, if not, the badge will not be displayed.

We can do it in the future though when we decide we need it (and we probably will for resume deposit, but who knows).

As for now I am resolving this conversation